dep/efsw/CMakeLists.txt

```diff
@@ -1,88 +1,112 @@
 if (BUILD_SHARED_LIBS)
   set(SRCS
-    src/efsw/Debug.cpp
-    src/efsw/DirectorySnapshot.cpp
-    src/efsw/DirectorySnapshotDiff.cpp
-    src/efsw/DirWatcherGeneric.cpp
-    src/efsw/FileInfo.cpp
-    src/efsw/FileSystem.cpp
-    src/efsw/FileWatcher.cpp
-    src/efsw/FileWatcherCWrapper.cpp
-    src/efsw/FileWatcherGeneric.cpp
-    src/efsw/FileWatcherImpl.cpp
-    src/efsw/Log.cpp
-    src/efsw/Mutex.cpp
-    src/efsw/String.cpp
-    src/efsw/System.cpp
-    src/efsw/Thread.cpp
-    src/efsw/Watcher.cpp
-    src/efsw/WatcherGeneric.cpp)
+          src/efsw/Debug.cpp
```

Comment:
are all these changes part of the upgraded library or are they TC-specific ones ?

Reply:
Looks like upstream changes to me https://bitbucket.org/SpartanJ/efsw/src/default/CMakeLists.txt

Reply:
it's overjustified compared to that repo.